account/user.go

```diff
@@ -174,3 +191,17 @@ func (m *GormUserRepository) Query(funcs ...func(*gorm.DB) *gorm.DB) ([]*User, e
 
 	return objs, nil
 }
+
+// UserFilterByID is a gorm filter for User ID.
+func UserFilterByID(userID uuid.UUID) func(db *gorm.DB) *gorm.DB {
+	return func(db *gorm.DB) *gorm.DB {
+		return db.Where("id = ?", userID)
+	}
+}
+
+// UserFilterByEmail is a gorm filter for User ID.
+func UserFilterByEmail(email string) func(db *gorm.DB) *gorm.DB {
```

Comment:
Yes @hectorj2f , otherwise the exposed `Query` function wouldn't be friendly to use in other packages.